juniper_official/Solutions/Traffic-Blackhole/host-interface-interrups.rule

```diff
@@ -0,0 +1,112 @@
+/*
+	* REQ: COL-1109
+	* Rule to collect Host-Interface Block CM Errors and Interrupts for Blackhole Detection RCA.
+*/
+healthbot {
+	topic hardware.fpc {
+		rule host-interface-interrupts {
+			keys [ fpc-name error-name ];
+			synopsis "Host-Interface Block CM Errors and Interrupts";
+			description "Host-Interface Block CM Errors and Interrupts";
+			sensor sensor-interrupts {
+				synopsis "Host-Interface Block CM Errors and Interrupts";
+				description "Host-Interface Block CM Errors and Interrupts";
+				open-config {
+					sensor-name /components/component/integrated-circuit/pipeline-counters/errors/host-interface-block/host-interface-block-error/;
```

Comment:
Change this /components/component to avoid duplicate subscriptions and reduce data from devices

Reply:
Changed path to: /components/component/